src/components/Overview/Overview.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactElement } from 'react';
+import { PageNode } from 'src/directory/directory';
+import { Card, Flex, Grid, View, Heading } from '@aws-amplify/ui-react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+type OverviewProps = {
+  childPageNodes: PageNode[];
+};
+
+export function Overview({ childPageNodes }: OverviewProps) {
+  const router = useRouter();
+  const currentPlatform = router.query.platform;
+
+  if (!childPageNodes) {
+    return <></>;
+  }
+
+  return (
+    <Grid className="overview">
```

Comment:
At the moment this grid will fill up the entire space of its parent, should we limit it when on big screens?

![image](https://github.com/aws-amplify/docs/assets/54393192/630bad78-5a4e-46b2-9a73-de9b5e8245c2)

Reply:
There should probably be some page container to limit the width of all the content on the page, especially since there will be a ToC. Would like to hear @hbuchel's thoughts here.